core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/kerberos/responses/KerberosV4Responses.java

```diff
@@ -0,0 +1,25 @@
+package com.sequenceiq.cloudbreak.api.endpoint.v4.kerberos.responses;
+
+import java.util.Set;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.google.common.collect.Sets;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.common.responses.GeneralCollectionV4Response;
+
+import io.swagger.annotations.ApiModel;
+
+@ApiModel
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
The @JsonIgnoreProperties and @JsonInclude annotations aren't in ClusterTemplateV4Responses. Are they necessary here, or necessary there?

Reply:
That was a copy-paste thing on my part, but there doesn't seem to be a lot of consistency between the objects. I'll remove the JsonIgnoreProperties and JsonInclude.